packages/vue/examples/search-showcase/voice-search/src/App.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div id="app">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.1/css/all.min.css"
+      integrity="sha512-gMjQeDaELJ0ryCI+FtItusU9MkAifCZcGq789FrzkiM49D8lbDhoaUaIX4ASU187wofMNlgBJ4ckbrXM9sE6Pg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer" >
+    <reactive-base
+      app="good-books-ds"
+      url="https://a03a1cb71321:75b6603d-9456-4a5a-af6b-a487b309eb61@appbase-demo-ansible-abxiydt-arc.searchbase.io"
+    >
+      <div class="container">
+        <div class="row">
+          <search-box
+            :data-field="['original_title', 'original_title.search']"
+            :urlparams="true"
+            :size="10"
+            :enable-popular-suggestions="true"
```

Comment:
@SavvyShah We can remove other props to lay emphasis on the `showVoiceSearch` prop